controllers/platform/services.go

```diff
@@ -99,27 +98,9 @@ func createDataIndexDeployment(ctx context.Context, client client.Client, platfo
 	liveProbe := readyProbe.DeepCopy()
 	liveProbe.ProbeHandler.HTTPGet.Path = common.QuarkusHealthPathLive
 	dataDeployContainer := &corev1.Container{
```

Comment:
@ricardozanini is the `data` prefix related to the data-index from the origin function purpose or serves a different meaning here? If it ties to `data-index`, can we omit it from the name of this variable to make this function more generic?

Reply:
Are you talking about the env var? I'm sure if I followed.

Reply:
i think he means, why is `data` part of the `dataDeployContainer` var. with @jordigilh's other changes, this could maybe be changed to `deployContainer`?

Reply:
+1

Reply:
Thank you @tchughesiv :-)